biowasm.json

```diff
@@ -356,6 +356,22 @@
 				}
 			]
 		},
+		{
+			"name": "hyphy",
+			"programs": ["hyphy"],
+			"description": "HyPhy",
```

Comment:
Could you add a brief description here, e.g. `Hypothesis testing using phylogenies`. That description will show up on the [package listing page](https://biowasm.com/cdn/v3)